app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt

```diff
@@ -134,60 +133,53 @@ fun PaymentSchedule(
+            )
+            Icon(
                 modifier = Modifier
-                    .fillMaxWidth(),
-                verticalAlignment = Alignment.CenterVertically,
-                horizontalArrangement = Arrangement.SpaceBetween
-            ) {
-                Text(
-                    modifier = Modifier.testTag(PaymentScheduleTestTags.PAYMENT_SCHEDULE_TITLE.name),
-                    text = stringResource(id = R.string.fpo_payment_schedule),
-                    style = typography.body2Medium,
-                )
-                Icon(
-                    modifier = Modifier
-                        .testTag(PaymentScheduleTestTags.EXPAND_ICON.name)
-                        .clickable { onExpandChange(!isExpanded) },
-                    painter =
-                    if (isExpanded) painterResource(id = R.drawable.ic_arrow_up) else painterResource(
-                        id = R.drawable.ic_arrow_down
-                    ),
-                    contentDescription = "Expand",
-                    tint = colors.textSecondary,
-                )
-            }
-
-            if (isExpanded) {
-                Spacer(modifier = Modifier.height(dimensions.paddingSmall))
-                paymentIncrements.forEach { paymentIncrement ->
-                    PaymentRow(paymentIncrement)
-                }
-                Spacer(modifier = Modifier.height(dimensions.paddingSmall))
-                KSClickableText(
-                    modifier = Modifier.testTag(PaymentScheduleTestTags.TERMS_OF_USE_TEXT.name),
-                    resourceId = R.string.fpo_terms_of_use,
-                    clickCallback = { onDisclaimerClicked.invoke(DisclaimerItems.TERMS) }
-                )
+                    .testTag(PaymentScheduleTestTags.EXPAND_ICON.name)
+                    .clickable { onExpandChange(!isExpanded) },
+                painter =
+                if (isExpanded) painterResource(id = R.drawable.ic_arrow_up) else painterResource(
+                    id = R.drawable.ic_arrow_down
+                ),
+                contentDescription = "Expand",
+                tint = colors.textSecondary,
+            )
+        }
+        if (isExpanded) {
+            Spacer(modifier = Modifier.height(dimensions.paddingSmall))
+            paymentIncrements.forEach { paymentIncrement ->
+                PaymentRow(paymentIncrement)
             }
+            Spacer(modifier = Modifier.height(dimensions.paddingSmall))
+            KSClickableText(
+                modifier = Modifier.testTag(PaymentScheduleTestTags.TERMS_OF_USE_TEXT.name),
+                resourceId = R.string.fpo_terms_of_use,
+                clickCallback = { onDisclaimerClicked.invoke(DisclaimerItems.TERMS) }
+            )
         }
     }
 }
 
 @Composable
 fun PaymentRow(paymentIncrement: PaymentIncrement) {
-    val formattedAmount = String.format(Locale.US, "%.2f", paymentIncrement.amount.amount.parseToDouble())
+    val formattedAmount =
```

Comment:
For formatting ammounts I will suggest to taking a look at `KSCurrency`, we want currency string appropriate to the user's locale

Reply:
The backend is going to provide the formatted amounts for us, we are just waiting for this piece to be finished. We actually don't show these amounts in the users currency, but in the project currency.